roles/compliance/tasks/main.yml

```
@@ -1,8 +1,11 @@
 ---
-- name: check for insights configuration
+- name: Check for Insights configuration
```

```suggestion
- name: Check for insights configuration
```
I would use lowercase here, as it is done in the insights_client tasks.

---

hm do you mean in `roles/insights_client/tasks/main.yml`? "Insights" is always title-cased in task names there

---

Oh you are right except for this [line](https://github.com/RedHatInsights/ansible-collections-insights/blob/master/roles/insights_client/tasks/main.yml#L59) :smile: 

Can you modify it as well?

---

sure! done